dclab/definitions/feat_const.py

```diff
@@ -118,6 +118,13 @@
     # Fun fact: If we had decided to compute it from the convex contour,
     # then we would have close to none pixelation effects ¯\_(ツ)_/¯.
     ["volume", "Volume [µm³]"],
+    ["area_um_raw", "Area [µm²] from a raw contour"],
```

Comment:
- Please insert the new features in alphabetical order in this list.
- Please replace "from a raw contour" with "of raw contour".

Reply:
I did it in a new commit. I hope you can see it?